governance_vote.py

```diff
@@ -1,13 +1,22 @@
 """Governance vote allow the collection of cosmos governance vote"""
 import json
+import os
 import time
 import traceback
 from dateutil import parser
 import requests
+from dotenv import load_dotenv
+from prometheus_client import start_http_server
 from utils import configure_logging
+from metrics import governance_votes_api_req_status_counter
+from constants.metrics_enum import MetricsNetworkStatus
+
 
 log = None
 
+load_dotenv()
+PROMOTEHEUS_PORT = os.getenv("PROMOTEHEUS_PORT", "9090")

```

Comment:
need to update this one since we don't use the port from the .env anymore

Reply:
i guess just remove it since i just saw we reference the config when starting the server http